shared/templates/sudo_defaults_option/bash.template

```diff
@@ -0,0 +1,10 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+if ! grep -P "^(?!#)[\s]*Defaults.*[\s]+{{{ OPTION }}}[\s]*.*$" /etc/sudoers; then
+    echo "Defaults {{{ OPTION }}}" >> /etc/sudoers
+    /usr/sbin/visudo -cf /etc/sudoers &> /dev/null || echo "Fail to validate /etc/sudoers with visudo"
```

Comment:
This is a smart move. What about taking it to a next level?
I was thinking of:
1. validation before the remediation
1. backup of the current sudoers file
1. remediation
1. revalidation - if not successful, revert, but end with an error
1. clean up the backup

Reply:
The template has been improved with suggested improvements